lib/cocina_display/utils.rb

```diff
@@ -35,16 +35,16 @@ def self.compact_and_join(values, delimiter: " ")
     #  cocina = { "structuredValue" => [{"value" => "foo"},  {"value" => "bar"}] }
     #  Utils.flatten_nested_values(cocina)
     #  #=> [{"value" => "foo"}, {"value" => "bar"}]
-    # @example parallel structured and simple values
-    #  cocina = { "parallelValue" => [{"value" => "foo" }, { "structuredValue" => [{"value" => "bar"},  {"value" => "baz"}] }] }
+    # @example nested structured and simple values
+    #  cocina = { "structuredValue" => [{"value" => "foo" }, { "structuredValue" => [{"value" => "bar"},  {"value" => "baz"}] }] }
     #  Utils.flatten_nested_values(cocina)
     #  #=> [{"value" => "foo"}, {"value" => "foo"}, {"value" => "baz"}]
     def self.flatten_nested_values(cocina, output = [], atomic_types: [])
       return [cocina] if cocina["value"].present?
       return [cocina] if atomic_types.include?(cocina["type"])
       return cocina.flat_map { |node| flatten_nested_values(node, output, atomic_types: atomic_types) } if cocina.is_a?(Array)
 
-      nested_values = Array(cocina["structuredValue"]) + Array(cocina["parallelValue"]) + Array(cocina["groupedValue"])
+      nested_values = Array(cocina["structuredValue"]) + Array(cocina["groupedValue"])
```

Comment:
We don't ever want to flatten `parallelValue`s anymore; the fields that can have them now have specific handling for when they occur.